test/cppsuite/test_harness/connection_manager.h

```diff
@@ -36,8 +36,6 @@ extern "C" {
 #include "wiredtiger.h"
 }
 
-#include "util/api_const.h"
-#include "util/debug_utils.h"
 #include "util/scoped_types.h"
```

Comment:
Do we still need this include here ?

Reply:
Yes, because `scoped_session` is used here. It compiles without the include because of the current order of includes. If somebody reorder includes it may lead to undefined type. So it's safer to have it here.